File: app/src/main/java/picker/ugurtekbas/com/DialTimePicker/ActivityAdapter.java
package picker.ugurtekbas.com.DialTimePicker;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

/**
 * Adapter for view pager.
 * @author Ugur Tekbas
 */
public class ActivityAdapter extends FragmentStatePagerAdapter{

    int[] layoutIDs = {R.layout.normal_picker, R.layout.ampm_picker};

    public ActivityAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        return MainFragment.newInstance(layoutIDs[position]);
    }

    @Override
    public int getCount() {
        return layoutIDs.length;
    }
}

File: app/src/main/java/io/flyingmongoose/EtzioDemo/ActivityAdapter.kt
package io.flyingmongoose.EtzioDemo

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentStatePagerAdapter

class ActivityAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

    var layoutIDs = intArrayOf(R.layout.normal_picker, R.layout.ampm_picker)

    override fun getItem(position: Int): Fragment {
        return MainFragment.newInstance(layoutIDs[position])
    }

    override fun getCount(): Int {
        return layoutIDs.size
    }
}
